hi-ml-cpath/src/health_cpath/preprocessing/create_tiles_dataset.py

```diff
@@ -87,36 +85,37 @@ def generate_tiles(slide_image: np.ndarray, tile_size: int, foreground_threshold
     return image_tiles, tile_locations, occupancies, n_discarded
 
 
-def get_tile_info(sample: Dict[SlideKey, Any], occupancy: float, tile_location: Sequence[int],
-                  rel_slide_dir: Path) -> Dict[TileKey, Any]:
+def get_tile_info(sample: Dict[SlideKey, Any], occupancy: float, tile_box: Box,
+                  rel_slide_dir: Path) -> dict:
     """Map slide information and tiling outputs into tile-specific information dictionary.
 
     :param sample: Slide dictionary.
     :param occupancy: Estimated tile foreground occuppancy.
-    :param tile_location: Tile XY coordinates.
+    :param tile_box: Tile box coordinates.
     :param rel_slide_dir: Directory where tiles are saved, relative to dataset root.
     :return: Tile information dictionary.
     """
     slide_id = sample[SlideKey.SLIDE_ID]
-    descriptor = get_tile_descriptor(tile_location)
+    descriptor = get_tile_descriptor(tile_box)
     rel_image_path = f"{rel_slide_dir}/{descriptor}.png"
 
     tile_info = {
         TileKey.SLIDE_ID: slide_id,
-        TileKey.TILE_ID: get_tile_id(slide_id, tile_location),
+        TileKey.TILE_ID: get_tile_id(slide_id, tile_box),
         TileKey.IMAGE: rel_image_path,
         TileKey.LABEL: sample[SlideKey.LABEL],
-        TileKey.TILE_X: tile_location[0],
-        TileKey.TILE_Y: tile_location[1],
+        TileKey.TILE_LEFT.value: tile_box.x,
+        TileKey.TILE_TOP.value: tile_box.y,
+        TileKey.TILE_RIGHT.value: tile_box.x + tile_box.w,
+        TileKey.TILE_BOTTOM.value: tile_box.y + tile_box.h,
         TileKey.OCCUPANCY: occupancy,
         TileKey.SLIDE_METADATA: {TileKey.from_slide_metadata_key(key): value
                                  for key, value in sample[SlideKey.METADATA].items()}
     }
-
     return tile_info
 
 
-def format_csv_row(tile_info: Dict[TileKey, Any], keys_to_save: Iterable[TileKey],
+def format_csv_row(tile_info: dict, keys_to_save: Iterable[TileKey],
```

Comment:
[Minor] Why change the type annotation here? I'd vote to keep `Dict[TileKey, Any]` explicitly for better type-checking.